stable/database/files/nuobackup

```diff
@@ -42,6 +44,64 @@ labels=""
 semaphore=""
 timeout=1800
 backup_root=$BACKUP_DIR
+wait_for_num_processes=0
+wait_timeout=300
+
+function await() {
+   local timeout=$1
+   shift
+   local end_time=$(( $(date -u +%s) + $timeout ))
+   local rc=0
+   while [ "$(date -u +%s)" -lt "$end_time" ]; do
+      eval "$*"
+      rc=$?
+      if [ $rc -eq 0 ]; then
+         return $rc
+      fi
+      sleep 1
+   done
+   return 124
+}
+
+function check_backup_processes() {
+   declare -a label_patterns=()
+   local all_process_labels=""
+
+   for ((i=0; i<${#labels_arr[@]}; i+=2)); do
+     label_patterns[${#label_patterns[@]}]="\"${labels_arr[i]}\"\s*:\s*\"${labels_arr[i+1]}\""
+   done
+
+   all_process_labels="$(nuocmd --api-server $NUOCMD_API_SERVER show database \
```

Comment:
`NUOCMD_API_SERVER` is used internally by `nuocmd` as the default. Why do we need to specify it explicitly?

Reply:
Good point. I used the same approach as in other `nuocmd` invocations. I see no point in specifying it explicitly unless @NikTJ777 say otherwise.